src/pages/Home/Home.styled.jsx

```diff
@@ -3,46 +3,114 @@ import { Link } from 'react-router-dom'
 import { Text as OriginalText } from '../../components'
 import text from '../../styles/text'
 
-export const Text = styled(OriginalText)`
-  ${[text.textCenter]}
-  ${props => props.title && text['7xl']}
-`
-
 export const Header = styled.div`
-  position: fixed;
+  display: flex;
+  flex-direction: column-reverse;
   width: 100%;
-  height: 55%;
-  background-color: var(--primary);
-  display: block;
-  margin: 0 auto;
+  height: 100%;
+  max-width: 100vw;
+  max-height: 100vh;
+
+  @media (min-width: 960px) {
+    flex-wrap: nowrap;
+    flex-direction: row;
+  }
+`
+export const HalfSection = styled.div`
+  ${props => `background-color: ${props.backgroundColor};`}
+  ${props => `justify-content: ${props.justifyContent};`}
+  ${props => `align-items: ${props.alignItems};`}
+  ${props => `height: ${props.mobileHeight};`}
+
+  display: flex;
+  flex-direction: column;
+  @media (min-width: 960px) {
+    width: 100%;
+    min-height: 100vh;
+  }
+`
+export const Logo = styled.img`
+  display: none;
+  max-width: 30%;
 
   @media (min-width: 960px) {
-    position: fixed;
-    margin-left: 50%;
-    width: 50%;
-    height: 100%;
-    background-color: var(--primary);
+    display: block;
+    margin-top: 8%;
+    margin-left: 8%;
   }
 `
 export const ManImage = styled.img`
   display: block;
-  margin: 0 auto;
-  height: 65vh;
+  width: 70%;
+  margin-top: 20%;
+
+  object-fit: contain;
+  position: absolute;
+  transform: translateY(-80%);
+
+  @media (min-width: 960px) {
+    transform: translateY(0%);
+    position: static;
+    align-self: center;
+    margin-top: 12%;
+    height: 68vh;
+  }
+
+  @media (min-height: 700px) and (min-width: 375px) and (max-width: 600px) {
+    margin-top: 25%;
+  }
+
+  @media (max-width: 320px) {
+    margin-top: 0;
+  }
+`
+export const TextContainer = styled.div`
+  transform: translateY(-80%);
+  max-width: 100%;
+
+  @media (min-width: 960px) {
+    transform: translateY(0%);
```

Comment:
Did you tried `transform: none;`, `transform: unset;` or `transform: revert;`?

Reply:
ready